neurodamus/metype.py

```diff
@@ -35,6 +48,61 @@ def connect2target(self, target_pp=None):
         """Connects empty cell to target"""
         return Nd.NetCon(self._cellref, target_pp)
 
+    @classmethod
+    def get_section_id(cls, cell, section):
```

Comment:
why not make this and `get_sec` free functions, since they don't apply to all BaseCells (ie: does EmptyCell need these methods)?